### ISSUE TITLE
Training Loss curve and log

### ISSUE THREAD

**Issue by yw155** [leoxiaobin/deep-high-resolution-net.pytorch#128]
Hi @leoxiaobin, could you provide the training loss record or log file? At least could you tell me the approximate loss value ranges? Thank you.
